packages/flutter/test/material/slider_test.dart

```diff
@@ -445,6 +445,67 @@ void main() {
     expect(tester.renderObject<RenderBox>(find.byType(Slider)).size, const Size(144.0 + 2.0 * 16.0, 32.0));
   });
 
+  testWidgets('discrete Slider respects textScaleFactor', (WidgetTester tester) async {
+    final Key sliderKey = new UniqueKey();
+    double value = 0.0;
+
+    Widget buildSlider({ double textScaleFactor }) {
+      return new Directionality(
+        textDirection: TextDirection.ltr,
+        child: new StatefulBuilder(
+          builder: (BuildContext context, StateSetter setState) {
+            return new MediaQuery(
+              data: new MediaQueryData(textScaleFactor: textScaleFactor),
+              child: new Material(
+                child: new Center(
+                  child: new OverflowBox(
+                    maxWidth: double.INFINITY,
+                    maxHeight: double.INFINITY,
+                    child: new Slider(
+                      key: sliderKey,
+                      min: 0.0,
+                      max: 100.0,
+                      divisions: 10,
+                      label: '${value.round()}',
+                      value: value,
+                      onChanged: (double newValue) {
+                        setState(() {
+                          value = newValue;
+                        });
+                      },
+                    ),
+                  ),
+                ),
+              ),
+            );
+          },
+        ),
+      );
+    }
+
+    await tester.pumpWidget(buildSlider(textScaleFactor: 1.0));
+    Offset center = tester.getCenter(find.byType(Slider));
+    TestGesture gesture = await tester.startGesture(center);
+    await gesture.moveBy(const Offset(10.0, 0.0));
+
+    expect(tester.renderObject(find.byType(Slider)),
+      paints..circle(radius: 6.0, x: 16.0, y: 44.0));
```

Comment:
style nit: this looks strange. I think our style is to put the closing param on the next line alined with expect.

Reply:
OK.  I think that's only when there's a newline after the open paren.  Anyhow, I fixed it up, I think.